src/components/FITBBlock.tsx

```diff
@@ -90,68 +37,99 @@ function Header({ hint }: HeaderProps) {
       >
         Fill in the Blank
       </Typography>
-      <HintDisplay hint={hint} />
+      <Box sx={{ display: 'flex', flexDirection: 'row' }}>
+        <ShowAnswerBtn onClick={handleShowAnswer}>Show Answer</ShowAnswerBtn>
+        <IconButton onClick={() => setShowHint(true)}>
+          <Lightbulb sx={{ color: '#04364a', fontSize: '2rem', m: 0 }} />
+        </IconButton>
+        <RestartAltIcon onClick={handleReset} />
+      </Box>
     </Box>
   )
 }
 
-// TODO(cam) migrate componenets to JSX.Elements instead of using strings here to support markdown & Latex
 interface FITBBlockProps {
-  question: string
   hint: string
+  children: ReactNode
 }
 
-export default function FITBBlock({ question, hint }: FITBBlockProps) {
-  const [submitted, setSubmitted] = useState<boolean>(false)
-  const [numAttempts, setNumAttempts] = useState<number>(3)
-  const [showAnswer, setShowAnswer] = useState<boolean>(false)
+export interface FITBState {
+  submitted: boolean
+  showAnswer: boolean
+}
 
-  const fitbForm = createFITBFormFromQuestionString(
-    question,
-    submitted,
-    showAnswer,
-  )
+export const FITBContext = createContext<FITBState>({
+  submitted: false,
+  showAnswer: false,
+})
+
+export default function FITBBlock({ hint, children }: FITBBlockProps) {
+  // only for the state we're passing down to the blanks
+  const defaultState: FITBState = { submitted: false, showAnswer: false }
+  const [state, setState] = useState<FITBState>(defaultState)
```

Comment:
nit: maybe we can use more specific names than "state". maybe something like defaultFitbState and fitbState